### ISSUE TITLE
Command line arrow should auto detect --baseUrl as --page and should not ask to start arrow_server

### ISSUE THREAD

**Issue by pranavparikh** [yahoo/arrow#254]
if I runt this command from arrow command line,
./node_modules/.bin/arrow ./arrow_files/basic_test/test-yahoo-title.js
--baseUrl=http://sports.yahoo.com --browser=firefox

It gives me error 

[2013-04-08 12:03:23.997] [ERROR] SeleniumDriver - Cannot load a local file
without arrow_server running:
[2013-04-08 12:03:23.997] [ERROR] Arrow - Cannot load a local file without
arrow_server running:
/but with this 

./node_modules/.bin/arrow ./arrow_files/basic_test/test-yahoo-title.js
--baseUrl=http://sports.yahoo.com --browser=firefox
--page=http://sports.yahoo.com

runs fine..

Basically, arrow should treat --baseUrl as --page in this case..
